modules/installation-complete-user-infra.adoc

```diff
@@ -192,7 +192,13 @@ $ oc get nodes -w
 ====
 If you have additional machine types such as infrastructure nodes, repeat the process for these types.
 ====
-endif::ibm-z,ibm-power[]
+endif::ibm-power[]
+
+ifdef::ibm-z[]
+. Additional steps are required to enable multipathing. Do not enable multipathing during installation.
++
+See the "Enabling multipathing with kernel arguments on {op-system}" in the _Installing on bare metal_ documentation for more information.
```

Comment:
@bobfuru Please replace reference with: 
See “Enabling multipathing with kernel arguments on {op-system}" in the _Post-installation machine configuration tasks_ documentation for more information.